packages/itwinui-css/src/table/table.scss

```diff
@@ -440,6 +442,7 @@
 
     svg:not(.iui-radial) {
       @include iui-icon-style('m');
+      fill: var(--_iui-table-cell-icon-fill);
```

Comment:
Test images did not change?

Reply:
We don't have any tests that catch this change.  I would need to add a new demo that tests combinations of status applied on rows & disabled / loading states.

Reply:
We probably should add tests for disabled state to catch any unwanted css changes in the future. It can be added to the same table - just add 3 more rows with status and disabled state.